network-area-diagram-viewer/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-parent</artifactId>
+        <version>4</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.example</groupId>
+    <artifactId>powsybl-network-area-diagram-viewer</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>11</java.version>
+        <franzxaver.version>0.1</franzxaver.version>
+        <javafx.version>17</javafx.version>
+        <slf4j.version>1.7.22</slf4j.version>
+        <junit.version>4.13.1</junit.version>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <powsybl.nad.version>0.4.0-SNAPSHOT</powsybl.nad.version>
+        <powsybl.core.version>4.8.0-SNAPSHOT</powsybl.core.version>
+        <powsybl.open.loadflow.version>0.18.0-SNAPSHOT</powsybl.open.loadflow.version>
```

Comment:
We want to test the latest changes of network-area-diagram, but not really the latest from powsybl-core and powsybl-open-loadflow. Hence, keep the snapshot version only for network-area-diagram